meetings/agenda.md

```diff
@@ -18,3 +18,20 @@ So questions are:
  - Should we abandon both packages? 
  - What components do still rely on the bridge?
  - Should we release a new version of these components relying on the bridge so they're not blocking upstream projects getting rid of the bridge?
+
+### Composer v2
+
+Composer [v2.0.0](https://github.com/composer/composer/releases/tag/2.0.0) was released on end of October 2020.
+Since then, the usage of `ocramius/package-versions` became obsolete and therefore, `composer/package-versions-deprecated` was already used to "override" `ocramius/package-versions`.
+
+Due to Marcos amazing idea of providing runtime information for composer dependencies, composer added this as a native feature with v2.0.0.
+
+However, we do still depend on the `composer/package-versions-deprecated` dependency as it does support PHP 8.0 and PHP 8.1 in the v1.x range while `ocramius/package-versions` would also require the v2+ version of it.
+In components which do support PHP 7.4 + PHP 8.0, we are forced to use the composer replacement.
+
+We do have two options as of november this year:
+
+1. remove PHP 7.4 support, remove `composer/package-versions-deprecated` and re-add `ocramius/package-versions` since it supports PHP 8.0 as of v2 
```

Comment:
This should probably happen anyway, as PHP 7.4 gets EOL'd

Reply:
Thats why I mention this. 
When we decide in the next TSC that we are fine with adding composer v2 dependencies, we can immediately drop the composer dependency at all and use composer v2 code.
If we decide to keep support of v1, we have to wait until PHP 7.4 gets EOL'd which is end of november.

Reply:
I see - in general, we don't need to actively work towards it, but rather let renovate figure out whether we can support/what.

Reminds me that I need to start enabling renovate for everything @_@